🌱 Add Script to Seed Consent Questions (#685)

mistryrn opened the pull request:
## Description of Changes

### Consent DAS
* Adds `seed-consent-questions.ts` script for populating the Consent DAS db with Consent Questions in deployed environments
* Updates `package.json` in Consent DAS to add `prisma:seed:questions` script which runs the new `seed-consent-questions.ts`

### Special Instructions
🚨  **Note: this script is intended to be run against fresh installs (empty databases) in DEPLOYED environments, and is meant to populate _just_ the consent questions. You do NOT need to run this in your local dev environment, especially if you have already run the regular `seed` script!** 🚨 

To run this script, from the `consent-das` root directory, run:
```bash
pnpm prisma:seed:questions
```

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [ ] New environment variables added to `.env.schema` files, `README.md`
- [x] Added copyrights to new files

mistryrn commented on apps/consent-das/prisma/seed/seed-consent-questions.ts:
```diff
@@ -0,0 +1,52 @@
+ */
+const seedConsentQuestions = async () => {
+	try {
+		logger.info('Seeding Consent Questions...');
+		await prisma.consentQuestion.createMany({
+			data: consentQuestions,
+			skipDuplicates: true,
+		});
+		logger.info('Seeding Consent Questions...COMPLETE');
```
Since this is meant to be run against a fresh install, I haven't done any clearing of existing consent questions here. Just seeding the consent questions.

anncatton replied:
Good idea. With `skipDuplicates` we won't end up with any unintended extra questions. Maybe add a note about the "fresh install" to your jsdoc.

mistryrn replied:
Updated, thanks! 🙇

anncatton reviewed (approved):
Looks good! Thanks for taking care of this. Should we maybe add a little README about this to the consent-das folder? Or record your PR description somewhere in the wiki (deployment instructions, not sure if we have a doc like that started)?

mistryrn commented:
Thanks for the feedback @anncatton – I updated the JSDoc and added an item to the `DAS Cleanup` ticket (https://github.com/OHCRN/platform/issues/662) for Consent DAS to add a README explaining the purpose and intended use of this script.

Also put together a draft "Deployment Instructions" doc in the wiki to capture this info too: https://wiki.oicr.on.ca/x/dgIcE

anncatton reviewed (approved):
thanks @mistryrn
